lib/api/redfish-1.0/systems.js

```diff
@@ -579,7 +579,7 @@ var doBootImage = controller(function(req,res) {
             graphName = 'Graph.InstallCentOS';
             graphOptions.defaults.osName = 'CentOS';
         } else if(payload.osName.indexOf('ESXi') !== -1) {
-            graphName = 'Graph.InstallEsx';
+            graphName = 'Graph.InstallESXi';
             graphOptions.defaults.osName = 'Esx';
```

Comment:
this is related to functions in codes, if changed, there will be function names and variables changed, avoid here, just change interface